types.go

```diff
@@ -105,6 +105,8 @@ type Source struct {
 
 	ContentTrust *ContentTrust `json:"content_trust,omitempty"`
 
+	Cosign *Cosign `json:"cosign,omitempty"`
```

Comment:
@taylorsilva Is an interesting point on the `get`. Wondering if I could do that in a separate PR? Correct me if I'm wrong, but this would allow us to verify that the `registry-image-resource` image that we are pulling is signed?

Reply:
Yes, it would! and totally fine to do that in another PR. Just wanted to determine if it made sense to keep the `cosign` field in the top-level source, or bring it down to the `params` used by the `out` command. Sounds like it makes sense to keep it up in `source` so someone can use it in a `get` step in a future PR.